spec/changelog.yml

```
@@ -18,6 +18,9 @@
     - description: Allow to override default deployment mode.
       type: enhancement
       link: https://github.com/elastic/package-spec/pull/854
+    - description: Add `terminate` processor to list pipeline spec.
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/847
```

Should it be PR 857 ?
```suggestion
      link: https://github.com/elastic/package-spec/pull/857
```